src/js/plugins/sync.js

```diff
@@ -29,22 +29,67 @@ ripe.Ripe.plugins.SyncPlugin.prototype.unregister = function(owner) {
     ripe.Ripe.plugins.Plugin.prototype.unregister.call(this, owner);
 };
 
+ripe.Ripe.plugins.SyncPlugin.prototype._buildRules = function(rules) {

```

Comment:
please comment this function and explain what it receives (as parameter) and what it returns

Reply:
name it `normalize_rules` instead